templates/hello_world_ts/Dockerfile

```diff
@@ -0,0 +1,40 @@
+# Dockerfile contains instructions how to build a Docker image that will contain
+# all the code and configuration needed to run your actor. For a full
+# Dockerfile reference, see https://docs.docker.com/engine/reference/builder/
+
+# First, specify the base Docker image. Apify provides the following base images
+# for your convenience:
+#  apify/actor-node-basic (Node.js on Alpine Linux, small and fast image)
+#  apify/actor-node-chrome (Node.js + Chrome on Debian)
+#  apify/actor-node-chrome-xvfb (Node.js + Chrome + Xvfb on Debian)
+# For more information, see https://apify.com/docs/actor#base-images
+# Note that you can use any other image from Docker Hub.
+FROM apify/actor-node-basic
+
+# Second, copy just package.json and package-lock.json since they are the only files
+# that affect NPM install in the next step
+COPY package*.json ./
+
+# Install NPM packages, skip optional and development dependencies to keep the
+# image small. Avoid logging too much and print the dependency tree for debugging
+RUN npm --quiet set progress=false \
```

Comment:
I think you need to build the code before you run it and it needs to be done during docker build.

Reply:
if you think it's better to add typescript as a dependency instead of a devDependency, I'll do it. but since you can't test locally without building it first, it makes sense to build then push/commit

Reply:
if the apify-cli supports using ts-node as the runtime instead of node, so that would work (didn't test to see)